src/SimpleFeatureToggle/FileFeatureLoadingStrategy.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using System.Diagnostics;
+
+namespace SimpleFeatureToggle
+{
+    public class FileFeatureLoadingStrategy : IFeatureLoadingStrategy
+    {
+        private readonly IFileReader _fileReader;
+        private string _fileName;
+
+        public FileFeatureLoadingStrategy(string fileName = null, IFileReader fileReader = null)
```

Comment:
Eek! I know you have added the `From` method, but I don't think `fileName` should be null 🤔.